templates/show-task.html

```diff
@@ -0,0 +1,133 @@
+{% extends 'base.html' %}
+
+{% block scripts %}
+    <script src="{% static 'js/csrf-token-functions.js' %}"></script>
+    <script src="{% static 'js/show-task.js' %}"></script>
+    <script type="text/javascript">
+    $(document).ready(function(){
+        tasks = {{tasks|safe}};
+        worklist_name = "{{worklist_name}}";
+        worklist_created_by = "{{worklist_created_by}}";
+        setTimeout(function(){
+        window.location.reload(true);
```

Comment:
I know this will change with React, but reloading the entire page is very annoying to the user.

I suppose we should really be serving the dynamic information as some JSON object in the server and doing XHRs here to fetch it the background, then updating only the relevant parts of the DOM -- and we'll need that server-side part with or without React, so let's do that now?

Reply:
In React, we'll have different components on the front end and to each component we'll be sending a JSON. In the backend we just need to break the views into smaller ones so that each one can correspond to a component. 
So, handling this right now will add work to the front end which won't be reused later. For now, I feel we can increase the time for refresh to make it less annoying. Because anyway, new front end will be there by July end.

But if you still feel it is an urgent requirement, I can complete it in a couple of days.

Reply:
Hmm, yeah I think it makes sense to go the JSON way for now to leave things in a generally clean state. Sure, some of that frontend code will go away if we go with React, but adding this intermediate state doesn't look like it would be a lot of work, and the view returning a proper JSON object is something we know we'll need either way, so might as well start figuring that out now and get proper UX on the client side from the start.

Reply:
Okay. Can I merge this request and raise that as a separate PR?

Reply:
Sure.